src/coreclr/md/inc/portablepdbmdi.h

```diff
@@ -69,6 +69,13 @@ DECLARE_INTERFACE_(IMetaDataEmit3, IMetaDataEmit2)
         USHORT      index,                  // [IN] Variable index (slot).
         char        *name,                  // [IN] Variable name.
         mdLocalVariable *locVarToken) PURE; // [OUT] Token of the defined variable.
+
```

Comment:
I have this feeling that I may need to introduce `IMetaDataEmit4`...

Reply:
@davidwrighton is that correct? Even though `IMetaDataEmit3` is only used for the portable PDB feature, if I want to add additional APIs, I will want to create a new one, i.e. `IMetaDataEmit4` ?

Reply:
Sadly, yes. Although if this is only for Ilasm... Give a different name.

Reply:
Ok, named it `IILAsmPortablePdbWriter`.